README.adoc

```diff
@@ -92,7 +92,7 @@ Now you are ready to https://bitcoinj.github.io/getting-started[follow the tutor
 
 Building apps with official releases of *bitcoinj* is covered in the https://bitcoinj.github.io/getting-started[tutorial].
 
-If you want to develop or test your app with a https://jitpack.io[Jitpack]-powered build of the latest `master` or `release-0.15` branch of *bitcoinj* follow the dynamically-generated instructions for that branch by following the correct link.
+If you want to develop or test your app with a https://jitpack.io[Jitpack]-powered build of the latest `master` or `v0.16.2` branch of *bitcoinj* follow the dynamically-generated instructions for that branch by following the correct link.
 
 * https://jitpack.io/#bitcoinj/bitcoinj/master-SNAPSHOT[master] branch
-* https://jitpack.io/#bitcoinj/bitcoinj/release-0.15-SNAPSHOT[release-0.15] branch
+* https://jitpack.io/#bitcoinj/bitcoinj/v0.16.2[v0.16.2] branch
```

Comment:
This should also be `release-0.16` see above.

Reply:
Makes sense. But I do not see a build in jitpack for release-0.16. Also tried pulling into my project but the artifact is not found as expected. I do see release-0.16-SNAPSHOT and that is getting pulled.

Reply:
Right. It needs to have `-SNAPSHOT` for jitpack to work correctly.